modules/account-lib/src/coin/near/keyPair.ts

```diff
@@ -1,6 +1,7 @@
-import { DefaultKeys, KeyPairOptions } from '../baseCoin/iface';
 import { Ed25519KeyPair } from '../baseCoin';
-import { AddressFormat } from '../baseCoin/enum';
+import { DefaultKeys, KeyPairOptions } from '../baseCoin/iface';
+import * as nearApi from 'near-api-js';
+import { toHex } from '../hbar/utils';
```

Comment:
we should move the `toHex` definition to a common file, the baseCoin's `baseUtils.ts` file looks like a better place to me